core/ui/EditToolbar/delete.tid

```diff
@@ -9,6 +9,9 @@ description: {{$:/language/Buttons/Delete/Hint}}
 {{$:/core/images/delete-button}}
 </$list>
 <$list filter="[<tv-config-toolbar-text>match[yes]]">
-<span class="tc-btn-text"><$text text={{$:/language/Buttons/Delete/Caption}}/></span>
+<span class="tc-btn-text">
+<$text text=" "/>
```

Comment:
We don't want to use whitespace for spacing. There are some dedicated CSS settings